lib/rspec/rails/matchers/active_job.rb

```diff
@@ -1,5 +1,6 @@
 require "active_job/base"
 require "active_job/arguments"
```

Comment:
We check if mocks are defined [here](https://github.com/rspec/rspec-rails/blob/609bb83a76c11c67076ff2619fd5f47edd6325ce/lib/rspec/rails/active_record.rb#L10). Should we do the same here instead of requiring what’s not used?

Reply:
The difference is here we are actually using it

Reply:
We’re only using it if verified partial doubles are checked, right? Which is not the case when RSpec is configured to mock with an alternative framework. Or am I missing something? It looks like before #2808 everything worked just fine, but apparently rspec-mocks were not required and RSpec::Mocks was not defined, and no problem.

Reply:
We don't want to skip the check just because someone is using another mocking framework

Reply:
If someone is using another framework, the result of the check will always be false. So it’s not that we skip it, we shortcut it.